packages/components/src/components/post-collapsible/post-collapsible.tsx

```diff
@@ -93,7 +87,11 @@ export class PostCollapsible {
 
   render() {
     return (
-      <Host data-version={version} tabindex={this.collapsed ? -1 : undefined}>
+      <Host
+        data-version={version}
+        tabindex={this.collapsed ? -1 : undefined}
+        style={IS_SERVER && this.collapsed ? (collapsedKeyframe as InlineStyles) : undefined}
```

Comment:
We could probably use the same approach on browser side: 
- always initialize the component using `collapsedKeyframe` in the `style` attribute
- use the `toggle` method only to toggle the collapsible once it's loaded